src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -0,0 +1,1161 @@
+import friendlyTime from 'friendly-time'
+import LearnerRatings from '../pages/courses/learner-ratings'
+import FiveStars from '../five-stars'
+import CommunityResource from 'components/community-resource'
+import TagList from './tag-list'
+import {useTheme} from 'next-themes'
+import ClosedCaptionIcon from '../icons/closed-captioning'
+import {HorizontalResourceCard} from '../card/horizontal-resource-card'
+import ExternalTrackedLink from 'components/external-tracked-link'
+import DialogButton from '../pages/courses/dialog-button'
+import MembershipDialogButton from '../pages/courses/membership-dialog-button'
+
+import LoginForm from 'pages/login'
+
+type CoursePageLayoutProps = {
+  lessons: any
+  course: any
+  ogImageUrl: string
+}
+
+type CollectionResource = {
+  title: string
+  duration: number
+  instructor: {
+    full_name: string
+  }
+  square_cover_url: string
+  image_url: string
+  path: string
+  slug: string
+  description: string
+}
+
+const LessonsList = ({
+  lessons,
+  completedLessonSlugs,
+}: {
+  lessons: any
+  completedLessonSlugs: any
+}) => {
+  return lessons.map((lesson: any, index: number) => {
+    const isComplete = completedLessonSlugs.includes(lesson.slug)
+    return (
+      <li key={lesson.slug}>
+        <div className="flex py-2 font-semibold leading-tight h-20">
+          <div className="flex items-center mr-2 space-x-2">
+            <div
+              className={`${
+                isComplete
+                  ? 'text-blue-600 dark:text-green-400'
+                  : 'text-gray-500 dark:text-gray-400'
+              } pt-px font-xs scale-75 font-normal w-4`}
+            >
+              {isComplete ? (
+                <CheckIcon className="w-6 h-6 -translate-x-2" />
+              ) : (
+                index + 1
+              )}
+            </div>
+            {lesson.icon_url && (
+              <div className="flex items-center flex-shrink-0 w-8">
+                <Image src={lesson.icon_url} width={24} height={24} />
+              </div>
+            )}
+          </div>
+          {lesson.path && (
+            <div className="flex flex-col ">
+              <div>
+                <Link href={lesson.path}>
+                  <a className="text-lg font-semibold hover:underline hover:text-blue-600 dark:text-gray-100">
+                    {lesson.title}
+                  </a>
+                </Link>
+              </div>
+              <div className="text-xs text-gray-700 dark:text-gray-500">
+                {convertTimeWithTitles(lesson.duration, {
+                  showSeconds: true,
+                })}
+              </div>
+            </div>
+          )}
+        </div>
+      </li>
+    )
+  })
+}
+
+const ModulesList = ({modules, render}: {modules: any; render: any}) => {
+  return modules.map((module: any) => {
+    return (
+      <div key={module.id}>
+        <h2 className="text-xl font-bold mt-4 mb-2">{module.title}</h2>
+        {(() => {
```

Comment:
Wrapped the switch in an IIFE since a statement is expected. We wanted the module title on the outside of the list so we thought this would be a better option. 

Though it might be better to have that h2 repeated in each case